workflow/src/main/java/com/synopsys/integration/alert/workflow/scheduled/frequency/ProcessingTask.java

```diff
@@ -58,29 +57,37 @@ public DateRange getDateRange() {
 
     @Override
     public void runTask() {
+        int jobCountByFrequency = jobAccessor.countJobsByFrequency(frequencyType.name());
+        if (jobCountByFrequency > 0) {
```

Comment:
As stated elsewhere, you should be able to make an Exists call instead of getting count then coparing

Reply:
done